android/src/main/java/com/amplitude/android/Amplitude.kt

```diff
@@ -60,7 +60,12 @@ open class Amplitude(
         if (this.configuration.offline != AndroidNetworkConnectivityCheckerPlugin.Disabled) {
             add(AndroidNetworkConnectivityCheckerPlugin())
         }
-        androidContextPlugin = AndroidContextPlugin()
+        androidContextPlugin = object : AndroidContextPlugin() {
```

Comment:
Do this for core as well

Reply:
Actually this isn't needed in core since core.ContextPlugin doesn't call setDeviceId()
https://github.com/amplitude/Amplitude-Kotlin/blob/232620cf75e4ece7718678ce6a4dd7553ad859b3/core/src/main/java/com/amplitude/core/platform/plugins/ContextPlugin.kt#L9

Reply:
Oh, right, core was doing it differently